File: kata-solutions/video-store-kata/video-store-java/src/main/java/RegularMovie.java
public class RegularMovie extends Movie {
  public RegularMovie(String title) {
    super(title);
  }

  @Override
  public double determineAmount(int daysRented) {
    double thisAmount = 2;
    if (daysRented > 2)
      thisAmount += (daysRented - 2) * 1.5;

    return thisAmount;
  }

  @Override
  public int determineFrequentRenterPoints(int daysRented) {
    return 1;
  }
}

File: kata-solutions/video-store-kata/video-store-python/src/regular_movie.py
from .movie import Movie


class RegularMovie(Movie):
    def __init__(self, title):
        super().__init__(title)

    def calculate_amount(self, days_rented) -> int:
        amount = 2
        if days_rented > 2:
            amount += (days_rented - 2) * 1.5
        return amount

    def calculate_frequent_renter_points(self, days_rented) -> int:
        return 1
